### ISSUE TITLE
Suggestion/Feature request - imbed target's unit-frame in the the main block

### ISSUE THREAD

**Issue by nobrains21** [Fiurs-Hearth/WIIIUI#1]
- It would be cool to have enemy unitframe imbeded in to the main block. I used my "incredible" ms-paint skills to make a mock-up version of what i mean by it and how i would suggest to implement it:

![Human_theme](https://user-images.githubusercontent.com/59528934/149619311-3430ae82-6c7c-49a8-aee9-12aebd644914.png)
This way you can have clean representation of your target and your "default" [4 by 3 +consumables] action bar, just like in Warcraft.
___

- You can also try add some graphic-panels above player/target unit-frames to display buffs and debuffs, or make them as a "titan panel type" of bar on top of the screen? A bar on top of the screen could represent the bar from Warcraft 3(the one where your resources are displayed), and feature mini-menu on it+bags and your money.

___
Thank you.

**Comment by Fiurs-Hearth** [comment]
Looks great!

I wont be making this anytime soon but its a great suggestion, will try and add this as an option in the menu when I have time for it.

Thanks for the suggestion.

**Comment by ratkosrb** [comment]
I don't think you can display the target's damage and armor, since those are "owner only" update fields.
